src/abbey/functions/in_group.rego

```diff
@@ -0,0 +1,9 @@
+package abbey.functions
+
+import future.keywords.if
+import future.keywords.in
+
+in_group(group_name) := true if {
+	some group in data.system.abbey.group_memberships
```

Comment:
@Koshroy-Abbey why the extra variable instead of `group_name in data.system.abbey.group_memberships`?